volatility3/framework/plugins/windows/cmdscan.py

```diff
@@ -100,7 +101,7 @@ def get_command_history(
             kernel_symbol_table_name: The name of the table containing the kernel symbols
             config_path: The config path where to find symbol files
             procs: list of process objects
```

Comment:
Seems odd to leave list capitalized if we're changing `an` to `An` just below it?